### ISSUE TITLE
setting ylim with coastline=True / without projection

### ISSUE THREAD

**Issue by ahuang11** [holoviz/hvplot#370]
This doesn't work
```
import xarray as xr
import hvplot.xarray
dz = xr.tutorial.open_dataset('air_temperature')
dz.hvplot('lon', 'lat', coastline=True).opts(ylim=(20, 30))
```
![image](https://user-images.githubusercontent.com/15331990/68491411-6ececb80-0210-11ea-9052-8e6f969f3556.png)

This works:
```
import xarray as xr
import hvplot.xarray
dz = xr.tutorial.open_dataset('air_temperature')
dz.hvplot('lon', 'lat', coastline=True, ylim=(0, 20))
```

This also works:
```
import xarray as xr
import hvplot.xarray
dz = xr.tutorial.open_dataset('air_temperature')
dz.hvplot('lon', 'lat', projection=ccrs.PlateCarree(), coastline=True).opts(ylim=(20, 30))
```

**Comment by maximlt** [comment]
I can reproduce.

**Comment by jsignell** [comment]
This will be fixed by the work in #1053
